### PR TITLE
Features/card functional

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 1]
```diff
@@ -71,7 +69,7 @@ public void iSetUpData(String data) {
     @When("I send the request")
     public void sentRequest() {
         response = requestManager.makeRequest();
-        String id = response.getStatusCode() == 200 ? response.body().jsonPath().get("id") : "";
+        String id = response.getStatusCode() == 200 ? response.body().jsonPath().get("[0].id") : "";
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java]: Please delete 0.

File: src/test/resources/feature/card/card_functional.feature [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      And  I verify the response schema with Card
+
+  @create-board @create-list @create-card @delete-card @delete-board
+  Scenario: Get the board a card is on
+    Given I create a GET request to /cards/{idCard}/board endpoint
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+
+#  @create-board @create-list @create-card @delete-card @delete-board
```
Comment (MadayAlcala) [inline src/test/resources/feature/card/card_functional.feature]: Please delete commented code.

File: src/test/resources/feature/card/card_functional.feature [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+#      And  I verify the response schema with Card
+
+  @create-board @create-list @create-card @create-checklist @delete-checklist @delete-card @delete-board
+  Scenario: Get the checklists on a card
+    Given I create a GET request to /cards/{idCard}/checklists endpoint
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+
+#  @create-board @create-list @create-card @delete-card @delete-board
```
Comment (MadayAlcala) [inline src/test/resources/feature/card/card_functional.feature]: Please delete commented code.

**Review by MadayAlcala — APPROVED** [review]
Good job!

File: src/test/resources/feature/card/card_functional.feature [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      And  I verify the response schema with Card
+
+  @create-board @create-list @create-card @delete-card @delete-board
+  Scenario: Get the stickers on a card
+    Given I create a GET request to /cards/{idCard}/stickers endpoint
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+
+# PUT FOR CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional.feature]: delete this comment line please.

File: src/test/resources/feature/card/card_functional.feature [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+          }
+        """
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+    Examples: new text valid for action
+    |text          |
+    |New commentary|
+
+#POST OF CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional.feature]: delete this comment line please.

File: src/test/resources/feature/card/card_functional.feature [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+                  """
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+    Examples: new name for checklist
+    |name               |
+    |New name CheckList |
+
+
+## DELETE OF CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional.feature]: delete this comment line please.

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java]: ok

File: src/test/resources/feature/card/card_functional.feature (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional.feature]: ok

File: src/test/resources/feature/card/card_functional.feature (hunk 3 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional.feature]: ok

File: src/test/resources/feature/card/card_functional.feature (hunk 4 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional.feature]: ok

File: src/test/resources/feature/card/card_functional.feature (hunk 5 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional.feature]: ok

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+                    }
+                  """
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+    Examples: new name for checklist
+    |name               |
+    |New name CheckList |
+    
+# DELETE OF CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: delete this comment line please.

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+          }
+        """
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+    Examples: new text valid for action
+    |text          |
+    |New commentary|
+
+# POST FOR CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: delete this comment line please.

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+      And  I verify the response schema with Card
+
+  @create-board @create-list @create-card @delete-card @delete-board
+  Scenario: Get the stickers on a card
+    Given I create a GET request to /cards/{idCard}/stickers endpoint
+    When I send the request
+    Then I get a 200 status code as response
+      And  I verify the response schema with Card
+
+# PUT FOR CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: delete this comment line please.

File: src/test/resources/feature/card/card_functional_positive.feature [hunk 10]
```diff
@@ -0,0 +1,147 @@
+Feature: Functional test for "card"
+
+#  GET FOR CARD
```
Comment (raulchoque-eng) [inline src/test/resources/feature/card/card_functional_positive.feature]: delete this comment line please.

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 7 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: ok

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 8 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: ok

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 9 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: ook

File: src/test/resources/feature/card/card_functional_positive.feature (hunk 10 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/feature/card/card_functional_positive.feature]: ok

**Review by raulchoque-eng — APPROVED** [review]

**Pull request merged**